vimeo-networking/src/main/java/com/vimeo/networking/callers/GetRequestCaller.java

```diff
@@ -0,0 +1,350 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Vimeo
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.vimeo.networking.callers;
+
+import com.vimeo.networking.VimeoClient;
+import com.vimeo.networking.VimeoClient.Caller;
+import com.vimeo.networking.VimeoService;
+import com.vimeo.networking.callbacks.VimeoCallback;
+import com.vimeo.networking.model.Category;
+import com.vimeo.networking.model.CategoryList;
+import com.vimeo.networking.model.Channel;
+import com.vimeo.networking.model.ChannelList;
+import com.vimeo.networking.model.Comment;
+import com.vimeo.networking.model.CommentList;
+import com.vimeo.networking.model.FeedList;
+import com.vimeo.networking.model.RecommendationList;
+import com.vimeo.networking.model.User;
+import com.vimeo.networking.model.UserList;
+import com.vimeo.networking.model.Video;
+import com.vimeo.networking.model.VideoList;
+import com.vimeo.networking.model.appconfiguration.AppConfiguration;
+import com.vimeo.networking.model.cinema.ProgramContentItemList;
+import com.vimeo.networking.model.notifications.NotificationList;
+import com.vimeo.networking.model.search.SearchResponse;
+import com.vimeo.networking.model.tvod.SeasonList;
+import com.vimeo.networking.model.tvod.TvodItem;
+import com.vimeo.networking.model.tvod.TvodList;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+import okhttp3.CacheControl;
+import retrofit2.Call;
+
+/**
+ * This is a collection of classes that should be used for get requests to
+ * {@link VimeoClient#getContent(String, CacheControl, VimeoCallback, String, Map, String, Caller)} or
+ * {@link VimeoClient#getContentSync(String, CacheControl, String, Map, String, Caller)}
+ * Created by rigbergh on 7/9/17.
+ */
+public final class GetRequestCaller {
```

Comment:
They probably have unused warnings because you are not viewing the library in the context of our app. For example, they shouldn't have warnings if you view the file in the context of the mobile project. Regardless, I will be creating a follow up ticket to write unit tests for the new networking API code that I have added. In general, I think we should be striving to create unit tests for our public APIs rather than suppressing the warnings whenever possible.